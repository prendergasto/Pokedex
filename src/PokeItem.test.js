import React from 'react';
import PokeItem from './Poke-item.js';
import renderer from "react-test-renderer";
test('renders PokeItem.js correctly', () => {
  const tree = renderer
  .create(<PokeItem data={[]}/>)
  .toJSON();
  expect(tree).toMatchSnapshot();
});

// test passing