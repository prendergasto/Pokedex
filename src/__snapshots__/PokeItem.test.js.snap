// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PokeItem.js correctly 1`] = `
<div
  id="pokemonContainer"
>
  <li
    id="pokemon-list"
  >
    <h1 />
    <img
      alt=""
      id="pokemon-images"
    />
    <h3>
      Type: 
    </h3>
    <h3>
      Ability: 
    </h3>
    <h3>
      Base experience: 
    </h3>
    <h3>
      Speed: 
    </h3>
  </li>
</div>
`;
